css/selectors/focus-visible-013.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>CSS Selectors Test: :focus-visible does not match after mouse focus even if previous focused element was matching :focus-visible</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<style>
+  @supports not (selector(:focus-visible)) {
+    :focus {
+      outline: red solid 5px;
+    }
+  }
+
+  #initial:focus-visible {
+    outline: green solid 5px;
+  }
+
+  #initial:focus:not(:focus-visible) {
+    outline: red solid 5px;
+  }
+
+  #target:focus-visible {
+    outline: red solid 5px;
+  }
+
+  #target:focus:not(:focus-visible) {
+    outline: green solid 5px;
+  }
+</style>
+
+<p>This test checks that if the active element matches ':focus-visible' and a mouse click causes focus to move elsewhere, the newly focused element should not match <code>:focus-visible</code>.</p>
+<ol id="instructions">
+  <li>If the user-agent does not claim to support the <code>:focus-visible</code> pseudo-class then SKIP this test.</li>
+  <li>Press the TAB key.</li>
+  <li>If the element that says "Initial" has a green outline, then the test rsult is a SUCCESS.</li>
```

Comment:
I would still prefer not to say the test is a success until we have actually tested the critical thing (i.e. that moving focus using the mouse from an element which matched `:focus-visible` causes the newly focused element *not* to match `:focus-visible` - and yes, this was a real bug that we temporarily had during our implementation process).

I don't want to block on it since this has dragged on so long, but I would prefer it if we could go back to the original behaviour/wording stating when the test is a FAILURE (analogous to an `assert()`), instead of having two SUCCESS states.

Reply:
Ok, I understand. I'll modify the instructions so SUCCESS is only at the end, but also explain intermediate FAILURES there. Thanks.